cmd/kpod/main.go

```diff
@@ -59,7 +59,7 @@ func main() {
 	}
 	app.Before = func(c *cli.Context) error {
 		logrus.SetLevel(logrus.ErrorLevel)
-		if c.GlobalBool("debug") {
+		if c.GlobalString("log-level") == "debug" {
```

Comment:
We should just set the log-level to whatever the user specified, not be specific to debug.

Reply:
logrus.SetLevel(c.GlobalString("log-level") )

Reply:
Crio has the following code.
```
		if loglevel := c.GlobalString("log-level"); loglevel != "" {
			level, err := logrus.ParseLevel(loglevel)
			if err != nil {
				return err
			}

			logrus.SetLevel(level)
		}
```

Reply:
fixed.